src/View/todo.md

```diff
@@ -0,0 +1,35 @@
+###Features:
+
+- [X] automatically generate json api for entity collections using dynamic schema
+- [X] automatically generate json api for single entities using dynamic schema
+- [X] allow overrides by creating manual schema
+- [X] add detector for `application/vnd.api+json` Accept request header 
+- [X] set viewVars using Crud listener configuration options
+- [X] respond with `application/vnd.api+json` Content-Type response header
+- [X] suggest neomerx composer package instead of require
+- [X] implement neomerx exception renderer for default errors
+- [X] implement neomerx exception renderer for validation errors
+- [X] add top-level `debug` and `query` nodes to exceptions
+- [X] add top-level `query` node to 200 responses
+- [ ] describe required custom route to support incoming requests to `self::SHOW_RELATED`
+- [ ] decode incoming jsonapi data to cake compatible format

```

Comment:
- renamed to `properly decode incoming jsonapi data` ;)
- personal todo list deleted from the repo